docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,277 @@
+string, one could add a past affiliation as precised below, but with a risk that Zenodo or tools using ``CITATION.cff``
+to make the ``codemeta.json`` does not understand it.
+
+ ::
+
+    authors:
+      - family-names: XXX
+        given-names: "YYYY Z."
+        affiliation: "Lab AA, XX; Past: Lab BB, YY"
+        email: yyyy.xxx@gammapy.org
+        orcid: "https://orcid.org/0000-0000-0000-0000"
+
+The ``CITATION.cff`` file will be used as the main file to be maintained, while the ``codemeta.json`` will be
+automatically created. For any release, the former should be carefully updated and systematically a review should be
+done by the dev team.
+
+In order to maintain updated data about authors for the LTS releases and LTS papers, a new file will be used with these
+data, ``LTS_AUTHORS.cff``. This file will be used to create the associated ``codemeta.json`` file.
+
+5. Possible implementation
+==========================
+
+Today, the tool offered by GitHub to sign the DCO is to add the extra parameter ``-s`` to each git commit. Users have to
+be aware of that and get used of it. For the reviewers, a github CI can be used to check quickly if a contributor has
+signed the DCO. This method will be used for the moment. If there is any other method that avoids this extra parameter,
+one will naturally consider it.
+
+In order to respect these rules, some automation is required to create the list of contributors. One could use
+the Python library `Tributors <https://con.github.io/tributors/>`_. This library uses the GitHub API to determine the
+list of contributors, which is written in the format associated with codemeta.json, citation.cff, zenodo.json. This kind
+of library allows then two requirements: retrieve automatically the list of contributors from GitHub and write the
+authors list in all the needed formats.
+
+One should note that the user profile of the GitHub platform does not contain a field associated with the ORCID
+identifier. However, one could also retrieve the ORCID identifier of a contributor with the ORCID API
+(e.g. `How-To-1 <https://stackoverflow.com/questions/71052912/extracting-credentials-from-orcid-seach-with-orcid-id-using-python>`_
+or `How-To-2 <https://info.orcid.org/documentation/api-tutorials/api-tutorial-searching-the-orcid-registry/#easy-faq-2532>`_).
+
+Even if this library is chosen, a specific Python script located in the main Gammapy repository had to be written such
+that the authors' order follows these rules. In this case, integration within a GitHub action could be possible.
+
+Finally, a second script should be settled in order to create the list of contributors in HTML format that should be
+inserted in the `team` section for our general web page. This script would read one of the metadata files (e.g.
+``citation.cff``) to create such list.
+
+Concerning the contributions to conferences, their handling should be realised properly by creating a dedicated
```

Comment:
This is vague. What does "properly realizing the handling" mean? What are we actually expecting here?

E.g. would:

> Conference contributions (Proceedings, Posters, ...) should be developed in dedicated repositories in the Gammapy Github organization?

be more clear of the intent?